pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -261,6 +262,43 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 		return reconcile.Result{}, errors.Wrap(err, "failed to save cluster config")
 	}
 
+	poolAndRadosNamespaceName := fmt.Sprintf("%s/%s", pool, getRadosNamespaceName(cephBlockPoolRadosNamespace))
+	if cephBlockPoolRadosNamespace.Spec.Mirroring.Enabled {
+		err = cephclient.EnableRBDRadosNamespaceMirroring(r.context, r.clusterInfo, poolAndRadosNamespaceName, cephBlockPoolRadosNamespace.Spec.Mirroring.RemoteNamespace, string(cephBlockPoolRadosNamespace.Spec.Mirroring.Mode))
```

Comment:
what happens if i pass the `RemoteNamespace` as a empty string? won't it try to enable mirroring between the namespace and the default namespace. as Ceph is not supporting this at the moment won't we need to block it as well?

Reply:
Correct